lambdas/services/authoriser_service.py

```diff
@@ -0,0 +1,106 @@
+import os
+import time
+
+from boto3.dynamodb.conditions import Key
+
+from enums.repository_role import RepositoryRole
+from services.dynamo_service import DynamoDBService
+from services.ssm_service import SSMService
+from services.token_service import TokenService
+from utils.audit_logging_setup import LoggingService
+from utils.exceptions import AuthorisationException
+from utils.request_context import request_context
+from utils.utilities import redact_id_to_last_4_chars
+
+logger = LoggingService(__name__)
+
+
+class AuthoriserService:
+    def __init__(
+        self,
+        path,
+        _http_verb,
```

Comment:
Any reason for the _ in this variable name?

Reply:
This only being used in a log really. and it was defined as _** is the original code.